addon/services/gwr.js

```diff
@@ -82,9 +82,7 @@ export default class GwrService extends Service {
       .replace(/\r?\n|\r/g, "");
 
     response = await this.authFetch.fetch(
-      query.page
-        ? `/${urlPath}?page=${query.page}&size=${this.pageSize}`
-        : `/${urlPath}`,
+      `/${urlPath}?page=${query?.page || 1}&size=${this.pageSize}`,
```

Comment:
Why not make the pagination optional? I think this wont work since we have no pagination on the error list in the quarterly closure tab.